src/lib/services/favorites-service.ts

```diff
@@ -0,0 +1,37 @@
+import { IUnleashConfig } from '../types/option';
+import { IUnleashStores } from '../types/stores';
+import { Logger } from '../logger';
+import {
+    IFavoriteFeatureKey,
+    IFavoriteFeaturesStore,
+} from '../types/stores/favorite-features';
+import { IFavoriteFeature } from '../types/favorites';
+
+export class FavoritesService {
```

Comment:
Do we need this service? It's a CRUD operation so controller can call the store directly. Is there anything this service adds on top of the store?

Reply:
I guess just best practice, aligns the rest of the codebase and disencourages to call stores directly from controller.